modules/ROOT/pages/prepare-raml-task.adoc

```diff
@@ -26,15 +27,21 @@ And then you can apply this trait in each individual operation like this:
     description: Gets a list of all the inventory products.
 ----
 
-. Click Policies on the API dashboard.
+== Getting Policies Specification Snippets
+
+. Click *Policies* on the API dashboard.
 +
-In the list of policies, the RAML snippet link contains the RAML code you need to add to the RAML.
+In the list of policies, the RAML snippet link contains the RAML trait or OAS components you need to add to the API specification.
 +
 image::raml-snippet.png[]
-. Click RAML Snippet.
+. Click *API Specification Snippet*.
+. Select the API type to match your specification.
+.. RAML 0.8 or RAML 1.0, or OAS 2.0 to match the API RAML version.
 +
 image::snippet-rate-limiting.png[]
-. Select RAML 0.8 or RAML 1.0 to match the API RAML version.
+.. Or OAS 2.0 to match the API OAS version.
```

Comment:
@fermujica, can we add a note saying that only since the following versions the policies contain the OAS snippets?

Rate limiting - SLA based | since v1.1.2
OAuth 2.0 access token enforcement using Mule OAuth provider  | since v1.1.3
OpenAM access token enforcement | since v1.1.3
OpenId Connect access token enforcement | since v1.1.3
OpenId Connect access token enforcement | since v1.1.3
Client ID enforcement | since v1.1.3

Reply:
@evangelina Sure. 
Can I assume that if the policies are not in this table, then they don't support OAS snippets?

Reply:
Let me know if this clarification works: fb8803f